.appveyor.yml

```diff
@@ -101,6 +103,8 @@ install:
   # Upgrade to the latest pip.
   - 'python -m pip install -U pip setuptools wheel'
 
+  - if [%INSTALL_PICKLE5%]==[1] echo pickle5 >> tools/ci/appveyor/requirements.txt
```

Comment:
I'm pretty sure our Appveyor webhook is now disabled in favor of Azure Pipelines for windows testing, so I'm not sure there's much value in modifying the appveyor yml file.

Reply:
Appveyor is still used for building wheels, let's not deprecate it just yet.